src/dodal/beamlines/i20_1.py

```diff
@@ -12,6 +16,12 @@
 from dodal.log import set_beamline as set_log_beamline
 from dodal.utils import BeamlinePrefix, get_beamline_name
 
+# introduces dependency on 'velocityprofile' package 
+from ophyd_async.epics.pmac import (
+    Pmac,
+    PmacMotor,
```

Comment:
Must: These don't exist in the latest `ophyd-async` (hence the test errors). Is there a specific `ophyd-async` branch that has these that you want to use? Or did they used to exist and been renamed?

Reply:
Yes, there are a bunch of changes in the parent https://github.com/bluesky/ophyd-async/issues/427 that we'll be making shortly that will feed into thos